src/views/users/const.js

```diff
@@ -100,3 +100,13 @@ export const userJSONSelectMeta = (vm, withoutOrgRole = false) => {
     }
   }
 }
+export const MFALevel = {
+  allUsers: 3,
+  onlyAdminUsers: 4
+}
+
+export const MFASystemSetting = {
+  notEnable: 0,
+  allUsers: 1,
+  onlyAdminUsers: 2
+}
```

Comment:
There does not appear to be any known issues with the provided code.

As of this date, the latest JavaScript version is likely >= v18. I suggest checking if you use Node.js 16+, which has slightly different features compared to older versions such as v14. In most cases, however, minor adjustments to existing scripts are unlikely but can still help enhance readability and efficiency:

- **Syntax**: Ensure that variable names and expressions within comments aren't too generic or ambiguous.
- **Flow** - Verify flow from line-to-line without unnecessary branches so there's no confusion during debugging.
- **Readability**: Consider renaming variables and/or commenting where needed for maintainability.
- **Type Checkings**: Use static type checkers like TypeScript’s for better error detection upfront during development.
- **Optimization**: If necessary, consider optimizing loops to decrease computational workload based on project needs, particularly if performance is an issue.